src/test/java/com/joestelmach/natty/AbstractTest.java

```diff
@@ -36,8 +37,8 @@ public void before() {
    * @param value
    * @return
    */
-  protected List<Date> parseCollection(String value) {
-    List<DateGroup> dateGroup = _parser.parse(value);
+  protected List<Date> parseCollection(String value, Date baseDate) {
```

Comment:
I think we should keep this consistent with the validate methods, passing the reference as the first parameter.